ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/StartupUtil.java

```diff
@@ -91,21 +92,19 @@ public static void setupInitialState(
     BeaconState initialState;
     if (startState != null) {
       try {
-        STATUS_LOG.log(
-            Level.INFO, "Loading initial state from " + startState, StatusLogger.Color.GREEN);
+        LOG.log(Level.INFO, "Loading initial state from " + startState);
         initialState = StartupUtil.loadBeaconStateFromFile(startState);
       } catch (final IOException e) {
         throw new IllegalStateException("Failed to load initial state", e);
       }
     } else {
-      STATUS_LOG.log(
+      LOG.log(
           Level.INFO,
           "Starting with mocked start interoperability mode with genesis time "
               + genesisTime
               + " and "
               + validatorKeyPairs.size()
-              + " validators",
-          Color.GREEN);
+              + " validators");
```

Comment:
These are both really important messages that should go to STDOUT. I'd probably keep the colours too.